index.html

```diff
@@ -32,42 +36,144 @@ <h1 class="logo">
+        <h3 class="project-name">Curios guy</h3>
+        <ul class="project-details">
+          <li class="lang">HTML/CSS</li>
+          <li class="lang">Ruby on Rails</li>
+          <li class="lang">JavaScript</li>
+        </ul>
+        <div class="button">
+          <button type="button" class="see-project">
+            See This project <img class="arrow" src="arrow.png" alt="arrow" />
+          </button>
+        </div>
+      </div>
+      <div class="project-info">
+        <img class="thumbnails" src="macpro.png" alt="project 1" />
+        <h3 class="project-name">IOS simulator</h3>
+        <ul class="project-details">
+          <li class="lang">HTML/CSS</li>
+          <li class="lang">Ruby on Rails</li>
+          <li class="lang">JavaScript</li>
+        </ul>
+        <div class="button">
+          <button type="button" class="see-project">
+            See This project <img class="arrow" src="arrow.png" alt="arrow" />
+          </button>
+        </div>
+      </div>
+      <div class="project-info">
+        <img class="thumbnails" src="apple.png" alt="project 1" />
+        <h3 class="project-name">Swift Application</h3>
+        <ul class="project-details">
+          <li class="lang">HTML/CSS</li>
+          <li class="lang">Ruby on Rails</li>
+          <li class="lang">JavaScript</li>
+        </ul>
+        <div class="button">
+          <button type="button" class="see-project">
+            See This project <img class="arrow" src="arrow.png" alt="arrow" />
+          </button>
+        </div>
+      </div>
+      <div class="project-info">
+        <img class="thumbnails" src="your.png" alt="project 1" />
+        <h3 class="project-name">Dating site</h3>
+        <ul class="project-details">
+          <li class="lang">HTML/CSS</li>
+          <li class="lang">Ruby on Rails</li>
+          <li class="lang">JavaScript</li>
+        </ul>
+        <div class="button">
+          <button type="button" class="see-project">
+            See This project <img class="arrow" src="arrow.png" alt="arrow" />
+          </button>
+        </div>
+      </div>
+    </section>
     <section class="About-section">
-      <div class="about">About</div>
+      <h3 class="about">About</h3>
       <p class="about-info">
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
-        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
-        veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea
-        eiusmod
+        Hello! I am a software developer! I can help you build a product,
+        feature or website. Taka a look of my works. If you like what you see
+        and have a project you need coded, don’t hesitate and contact me.
       </p>
-      <button type="button" class="cv">
-        <img src="Group86.png" alt="yes" />
+      <button type="button" class="connect">
+        Get my CV
+        <img src="dont.png" alt="yes" />
       </button>
```

Comment:
- [ ] Please make sure this button contains 100% of the width and matches with the Figma design

### Your design

![image](https://user-images.githubusercontent.com/83803414/165038209-ee186983-96f3-4c78-af6e-5ebd6c04270e.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/165038225-9527c976-9d84-4ede-a312-f5182568e55d.png)